nipype/interfaces/ants/utils.py

```diff
@@ -357,3 +357,60 @@ class ComposeMultiTransform(ANTSCommand):
     _cmd = 'ComposeMultiTransform'
     input_spec = ComposeMultiTransformInputSpec
     output_spec = ComposeMultiTransformOutputSpec
+
+
+class LabelGeometryInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(
+        3,
+        2,
+        argstr='%d',
+        usedefault=True,
+        position=0,
+        desc='image dimension (2 or 3)')
+    label_image = File(
+        argstr='%s',
+        position=1,
+        mandatory=True,
+        desc='label image to use for extracting geometry measures')
+    intensity_image = File(
+        value='[]',
+        exists=True,
```

Comment:
`exists=True` seems weird, given a default value of `'[]'`. I guess it applies only to non-default values?

Reply:
`exists` only applies to non-default values (could be considered a bug). i'm taking advantage of that here, without writing a `_format_arg` function, but something to keep in mind.